charts/dsp-atlantis/templates/cleanup/cronjob.yaml

```diff
@@ -0,0 +1,50 @@
+{{- if .Values.cleanup.enabled }}
+apiVersion: batch/v1beta1
+kind: CronJob
+metadata:
+  name: atlantis-cleanup-cronjob
+  labels: 
+    {{- include "dsp-atlantis.labels" . | nindent 4 }}
+spec:
+  schedule: "30 9 * * 1" # Run every Monday at 9:30 am UTC / 4:30 am EST
+  concurrencyPolicy: Replace
+  jobTemplate:
+    metadata:
+      labels: 
+        {{- include "dsp-atlantis.labels" . | nindent 8 }}
+    spec:
+      activeDeadlineSeconds: {{ .Values.cleanup.timeoutSeconds }}
+      template:
+        metadata:
+          labels:
+            {{- include "dsp-atlantis.labels" . | nindent 12 }}
+        spec:
+          serviceAccountName: {{ .Values.cleanup.saName }}
+          containers:
+          - name: atlantis-cleanup-container
```

Comment:
I don't think you need to specify `container` in the name here. Just `atlantis-cleanup` is fine